File: Python_Codes/code2.py
def menu():
    print("----------- MENU -----------")
    print("c: Count up to number 1000")
    print("h: Say hello")
    print("q: Quit")

def condition(user_input):
    if user_input == "q":
        print("Program finished")
        return False
    elif user_input == "h":
        print("Hello!")
    elif user_input == "c":
        print("Counting up to number 1000:")
        for i in range(1001):
            print(f"Number: {i}")
    else:
        print("Unknown option. Try again")
    return True

def main():
    user_input = ""
    print("Initiating program:")
    cond = True
    while cond:
        menu()
        user_input = input("Select an option: ")
        cond = condition(user_input)

if __name__ == "__main__":
    main()

File: C++_Codes/code2.cpp
#include <iostream>
#include <string>

using namespace std;

void menu(){
    cout << "----------- MENU -----------\n";
    cout << "c: Count up to number 1000\n";
    cout << "h: Say hello\n";
    cout << "q: Quit\n";
}

bool condition(string input){
    if (input == "q"){
        cout << "Program finished";
        return false;
    }
    else if( input == "h"){
        cout << "Hello!\n";
    }
    else if( input == "c"){
        cout << "Counting up to number 1000:\n";
        for(int i {0};i<=1000;i++){
            cout << "Number: " << i << endl;
        }
    }
    else {
        cout << "Unknown option. Try again\n";
    }
    return true;
    }


int main() {
    string input {""};
    cout << "Initiating program':\n";
    bool cond {true};
    while (cond)
    {
        menu();
        cout << "Select an option: "; 
        cin >> input;
        cond = condition(input);
    }

    return 0;
}